src/main.rs

```diff
@@ -0,0 +1,72 @@
+// EasySplash - tool for animated splash screens
+// Copyright (C) 2020  O.S. Systems Software LTDA.
+//
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+mod animation;
+mod gstreamer;
+mod message;
+
+use crate::{animation::Animation, message::Message};
+
+use argh::FromArgs;
+use env_logger::Env;
+use log::info;
+use std::path::PathBuf;
+
+/// EasySplash offers a convenient boot splash for Embedded Linux devices,
+/// focusing of simplicity and easy to use.
+#[derive(FromArgs)]
+struct CmdLine {
+    #[argh(subcommand)]
+    inner: Commands,
+}
+
+#[derive(FromArgs)]
+#[argh(subcommand)]
+enum Commands {
+    Open(Open),
+    Client(Client),
+}
+
+/// open the render with the specific animation
+#[derive(FromArgs)]
+#[argh(subcommand, name = "open")]
+struct Open {
+    /// path to load the animation
+    #[argh(positional)]
+    path: PathBuf,
+}
+
+/// control the render from the user space
+#[derive(FromArgs)]
+#[argh(subcommand, name = "client")]
+struct Client {
+    /// stop the render as soon as possible
+    #[argh(switch)]
+    stop: bool,
+}
+
+#[async_std::main]
+async fn main() -> Result<(), anyhow::Error> {
+    // Initilize the logger (default to info level)
+    env_logger::from_env(Env::default().default_filter_or("info")).init();
```

Comment:
Having the log controlled by the command line options itself would be more user-friendly.

Reply:
I converted it to use `simple_logging` and added a command line for both open and client sub commands.